### PR TITLE
add browserless loader

### PR DESCRIPTION
**Pull request by jagilley**
# Browserless

Added support for Browserless' `/content` endpoint as a document loader.

### About Browserless

Browserless is a cloud service that provides access to headless Chrome browsers via a REST API. It allows developers to automate Chromium in a serverless fashion without having to configure and maintain their own Chrome infrastructure.

### CODE REVIEW

File: docs/extras/modules/data_connection/document_loaders/integrations/browserless.ipynb
Comment (jagilley) [reply docs/extras/modules/data_connection/document_loaders/integrations/browserless.ipynb:1]: You got it @baskaryan

Comment (rlancemartin) [reply docs/extras/modules/data_connection/document_loaders/integrations/browserless.ipynb:1]: +1 of to @baskaryan comment, and please add # Browserless header.

**Review by rlancemartin — APPROVED** [review]
lgtm pending addition of context in the notebook as @baskaryan said.

also, would be cool to update the [recursive_url_loader](https://github.com/hwchase17/langchain/blob/00c466627a1d80c92376e9748f9557111c2030af/langchain/document_loaders/recursive_url_loader.py#L97) w/ option to supply different page loaders. 

i can take that up in a follow-up PR once this merges.

File: docs/extras/modules/data_connection/document_loaders/integrations/browserless.ipynb
Comment (jagilley) [reply docs/extras/modules/data_connection/document_loaders/integrations/browserless.ipynb:1]: Ohh I see, a header in the notebook itself

File: langchain/document_loaders/browserless.py
Comment (rlancemartin) [inline langchain/document_loaders/browserless.py:13]: make `urls: Union[str, List[str]],` for [consistency](https://github.com/hwchase17/langchain/blob/8741e55e7c5ce8e389081f52dd949781269a6725/langchain/document_loaders/web_base.py#L61C11-L61C11).

Comment (jagilley) [reply langchain/document_loaders/browserless.py:13]: Done @rlancemartin

Comment (rlancemartin) [reply langchain/document_loaders/browserless.py:13]: thanks! testing now.

**Pull request merged**
